tests_with_redis.sh

```diff
@@ -29,6 +29,7 @@ then
 elif [ "$1" == "integrationtest" ]
 then
   pytest -m 'not unittest and not noauth'
+  run_tests_notauth
```

Comment:
I'd rename `run_tests_notauth` to `run_tests_noauth` for better readability, if possible.